docs/operator-manual/high_availability.md

```diff
@@ -80,6 +80,8 @@ spec:
           value: "2"
 ```
 
+* The shard distribution algorthim of the `argocd-application-controller` can be set by using the `--sharding-method` parameter. Supported sharding methods are : [legacy (default), round-robin]. `legacy` mode uses an `uid` based distribution (non-uniform). `roudn-robin` uses an equal distribution accross all shards.
```

Comment:
```suggestion
* The shard distribution algorithm of the `argocd-application-controller` can be set by using the `--sharding-method` parameter. Supported sharding methods are : [legacy (default), round-robin]. `legacy` mode uses an `uid` based distribution (non-uniform). `round-robin` uses an equal distribution across all shards.

!!! warning "Alpha Feature"
    Round-robin is an experimental feature. Reshuffling is known to occur in certain scenarios with cluster removal. If the cluster at rank-0 is removed, it will trigger all clusters to reshuffle across shards.
```

Suggesting some language for the warning and fixing some spelling errors.

Reply:
@todaywasawesome thanks, I have added this part and fixed the typo.